content/2022/portuguese/_index.md

```diff
@@ -25,3 +28,7 @@ tagline_suffix: |
 {{< events/program-committee event="portuguese" year="2022" source="committee" subpage="program-committee" >}}
 We are proud to say that the sessions and keynotes for JakartaOne Livestream Event are chosen by an independent program committee made up of volunteers from the Jakarta&reg; EE and Cloud Native Java community!
 {{</ events/program-committee >}}
+
```

Comment:
we usually put the agenda above the program committee section. Only issue I see, so once you fix it on this and pt page we should be good

Reply:
Updated :)